### PR TITLE
Add drivers for both Ricoh and Canon printers.

### PR DESCRIPTION
**Pull request by renard**
Supported printers:
- Ricoh Aficio SP C240DN
- Canon UFR II series

### CODE REVIEW

File: Casks/canon-driver-ufr-ii.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Install via: brew cask canon-driver-ufr-ii
```
Comment (vitorgalvao) [inline Casks/canon-driver-ufr-ii.rb]: The comment is useless, and must be removed.

File: Casks/ricoh-driver-aficio-sp-c240dn.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Install via: brew cask ricoh-driver-aficio-sp-c240dn
```
Comment (vitorgalvao) [inline Casks/ricoh-driver-aficio-sp-c240dn.rb]: Same here.

File: Casks/ricoh-driver-aficio-sp-c240dn.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+cask :v1 => 'ricoh-driver-aficio-sp-c240dn' do
+  version '2.022.02'
+  sha256 'ee97bb2775f62b69785996b475ed2b599872d618c4630f543a2603ef75f73c04'
+
+  url 'http://support.ricoh.com/bb/pub_e/dr_ut_e/0001273/0001273748/V202/r69453Lf.dmg'
+  name 'Ricoh Aficio SP C240DN driver'
+  homepage 'http://support.ricoh.com/bb/html/dr_ut_e/ra/model/spc240/spc240fr.htm'
+  license :closed 
+
+  pkg  'Ricoh Aficio SP C240DN.pkg'
```
Comment (vitorgalvao) [inline Casks/ricoh-driver-aficio-sp-c240dn.rb]: Should be an empty line under this.

File: Casks/canon-driver-ufr-ii.rb [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+cask :v1 => 'canon-driver-ufr-ii' do
+  version 'v10.8.1'
+  sha256 'f4935b6c787648fb71df0bfff02d1a021faac85e8fa0e752ba70c0fa610ed229'
+
+  url 'http://files.canon-europe.com/files/soft46642/software/o1591mux_m_ufr1081.dmg'
+  name 'Canon UFR II / UFR II LT Printer Driver'
+  homepage 'http://fr.software.canon-europe.com/products/0010767.asp'
+  license :closed 
+
+  pkg  'Office/UFRII_LT_LIPS_LX_Installer.pkg'
```
Comment (vitorgalvao) [inline Casks/canon-driver-ufr-ii.rb]: Should be an empty line under this.

File: Casks/canon-driver-ufr-ii.rb [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  uninstall :pkgutil => [
```
Comment (vitorgalvao) [inline Casks/canon-driver-ufr-ii.rb]: Are you sure you aren’t removing too much? Looks like you’re removing unrelated stuff that might break other drivers.

Comment (renard) [inline Casks/canon-driver-ufr-ii.rb]: This driver installs a lot of packages. The result here is from a diff before and after installation.

Comment (renard) [inline Casks/canon-driver-ufr-ii.rb]: Anyway I just realize this is duplicated with canon-imagerunner-printer-driver-ufrii

Comment (vitorgalvao) [inline Casks/canon-driver-ufr-ii.rb]: But stuff like `jp.co.canon.CUPSPrinter.core.pkg` and `jp.co.canon.CUPSPrinter.profiles.CoreICC.pkg` seems to be something shared among other drivers. Not sure we should keep it, but we can try and see if it causes trouble. Either way, it is too verbose, simply [do this](https://github.com/caskroom/homebrew-cask/blob/00fdad2904f76f8204160383298a94f5f620581d/Casks/canon-imagerunner-printer-driver-ufrii.rb#L13).

Comment (vitorgalvao) [inline Casks/canon-driver-ufr-ii.rb]: The please remove this one from the PR.

**Comment by renard** [comment]
Please discard this PR it is buggy and badly done. I will create a new one
